Dockerfile

```diff
@@ -41,9 +41,10 @@ RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o /src/${PROJECT_NAME} \
 # =============================================================================
 FROM ${DOCKER_REGISTRY:+$DOCKER_REGISTRY/}alpine:${ALPINE_VERSION} AS final
 
-RUN apk add --update \
+RUN apk add --upgrade \
```

Comment:
I think we should keep the `update` too.
The recommendation is to run the update first (it updates the list of available packages) and then the upgrade, as in
```
apk update
apk add --upgrade busybox 
```
See the documentation: https://wiki.alpinelinux.org/wiki/Package_management#Upgrade_a_Running_System

Reply:
In Dockerfile, it would translate to `RUN apk update && apk add --upgrade \`

Reply:
Thanks @jkhalack for pointing it out. Will fix this